assets/js/editor-new.js

```diff
@@ -391,8 +391,8 @@
       imageUploadFunction: uploadImage,
       imageTexts: {
         sbInit: 'Joindre des images par glisser-déposer ou coller depuis le presse-papiers.',
-        sbOnDragEnter: 'Déposer l"image pour l"envoyer dans votre galérie',
-        sbOnDrop: 'Téléchargement d"images #images_names#',
+        sbOnDragEnter: 'Déposez l\'image pour l\'envoyer dans votre galérie',
```

Comment:
T'aimes vraiment cet accent ? :p

```suggestion
        sbOnDragEnter: 'Déposez l\'image pour l\'envoyer dans votre galerie.',
```

Reply:
Mais j'ai pas les yeux en face des trous c'est dingue !